README.md

```diff
@@ -153,27 +152,26 @@ import Polysemy.Resource
 data CustomException = ThisException | ThatException deriving Show
 
 program :: Members '[Resource, Teletype, Error CustomException] r => Sem r ()
-program = catch @CustomException work $ \e -> writeTTY ("Caught " ++ show e)
-  where work = bracket (readTTY) (const $ writeTTY "exiting bracket") $ \input -> do
-          writeTTY "entering bracket"
-          case input of
-            "explode"     -> throw ThisException
-            "weird stuff" -> writeTTY input >> throw ThatException
-            _             -> writeTTY input >> writeTTY "no exceptions"
+program = catch @CustomException work \e -> writeTTY $ "Caught " ++ show e
+ where
+  work = bracket (readTTY) (const $ writeTTY "exiting bracket") \input -> do
+    writeTTY "entering bracket"
+    case input of
+      "explode"     -> throw ThisException
+      "weird stuff" -> writeTTY input *> throw ThatException
+      _             -> writeTTY input *> writeTTY "no exceptions"
 
 main :: IO (Either CustomException ())
-main =
-    runFinal
-  . embedToFinal @IO
-  . resourceToIOFinal
-  . errorToIOFinal @CustomException
-  . teletypeToIO
-  $ program
+main = runFinal
```

Comment:
I think this
```haskell
main
  = runFinal
  . embedToFinal @IO
  ...
```
would be more consistent with the other code above (`runTeletypePure`)